recipe_box/src/main/java/com/spr/recipe_box/RecipeService.java

```diff
@@ -17,7 +18,7 @@
     private static List<Recipe> recipeList = new ArrayList<>();
 
     public List<Recipe> findById(String recipe_id){
-            String url  = "https://api.bigoven.com/recipe/"+recipe_id+"?api_key=glFUKikehWjLW900etpS564VgIzOWSW5";
+            String url  = "https://api.bigoven.com/recipe/"+recipe_id+ Keys.BIG_OVEN_KEY;

```

Comment:
Can we change this to where: 
BIG_OVEN_KEY = glFUKikehWjLW900etpS564VgIzOWSW5
and keep the  `?api_key` out here?

Reply:
Also, if we are changing the end points to constants can we add one for  `https://api.bigoven.com/`?

Reply:
I completely agree I'll get this switched over!